BankSDK/GiniBankSDKExample/GiniBankSDKExample/AppCoordinator.swift

```diff
@@ -160,7 +161,37 @@ final class AppCoordinator: Coordinator {
                         for: [GiniCapturePage(document: document, error: nil)])
                 } catch  {
                     self.rootViewController.showErrorDialog(for: error, positiveAction: nil)
-                }
+                }            // When a document is imported with "Open with", a dialog allowing to choose between both APIs
+                // is shown in the main screen. Therefore it needs to go to the main screen if it is not there yet.
+
+            }
+        }
+    }
+    
+    func processExternalDocumentFromPhotos(withUrl url: URL, sourceApplication: String?) {
+        let captureConfiguration = configuration.captureConfiguration()
+        // 1. Build the document
+        let documentBuilder = GiniCaptureDocumentBuilder(documentSource: .appName(name: sourceApplication))
+        documentBuilder.importMethod = .openWith
+        
+        // Get saved imageData which was save in the extension from the group extension UserDefaults
+        // Build document with imageData because it's not possible to do it for the url from Photos
+        if let userDefaults = UserDefaults(suiteName: groupName) {
+            if let data = userDefaults.value(forKey: imageDataKey) {
+             if let doc = documentBuilder.build(with: data as! Data, fileName: "image") {
```

Comment:
[Suggestion] Personally I prefer one if with all the conditions since we are not doing anything special for any other case. But if you feel that is not the right moment to do and discuss this please ignore this.